package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/exaptis/asciidoctor-loader",
   "dependencies": {
-    "asciidoctor.js": "~1.5.5-2",
+    "asciidoctor.js": "1.5.5-4",
```

Comment:
Ok, valid point! Currently there is no testing procedure in place. Off-topic question, are experiencing any noticeable slowdowns during your webpack build when using this loader?